### DISCUSSION TITLE
Creation of Mia-Platform Nexus pull secret in the K8S namespace

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#222] [Q&A] [answered]
### Which section?

Console

### Which is the version?

10.6.0

### Description

When I create a new project from my self-hosted Console, I would like to have the console automatically create the pull secret in the K8S namespace to pull the Mia-Platform plugin from Mia-Platform Nexus. Is it possible?

### Actual Outcome

When I create a new project I have to manually insert the Mia-Platform Nexus pull secret in the K8S namespace

### Expected Outcome

I qould like to have the secret automatically created by the Console

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, are you using [mlp](github.com/mia-platform/mlp)? If so you can use it to create a secret for you!

Here you can find an example of the configuration for it to work in your deployment pipeline to ensure the secret exists: https://docs.mia-platform.eu/docs/runtime_suite_tools/mlp/generate

**Comment by lucascanna** [comment]
Unfortunatly I am not using MLP at the moment. Do you have any alternatives to suggest?

I am considering to use the upcoming feature of the Console that will call a custom webhook at project creation.
What do you think about it?
